src/shared/components/in-app-search/InAppSearch.tsx

```diff
@@ -137,8 +148,12 @@ const InAppSearch = (props: Props) => {
           </div>
         )}
       </div>
-      {searchResult && (
-        <InAppSearchMatches {...searchResult} app={app} mode={mode} />
+      {isLoading ? (
+        <CircleLoader className={styles.spinner} size="small" />
```

Comment:
CircleLoader serves the purpose but would a LoadingButton be better here?

Reply:
I discussed this with Andrea when I was considering what to do with the button — whether to use a loading button or to disable it on submit. Andrea preferred a spinner in the results section. As for the `size` prop, she also said she would like us to make the spinner in the interstitial smaller.

https://user-images.githubusercontent.com/6834224/158917844-4a999835-bc77-425b-a5a3-0fb818bfc688.mp4

She said that, in general, that she expected that we would need a larger spinner and a smaller spinner, which is why I created two default spinner sizes. We can ask her again...

Reply:
ok no problem.